src/server-entry.js

```diff
@@ -90,7 +92,8 @@ export default context => {
 		// render content for template
 		context.renderedConfig = renderGlobals({ __KV_CONFIG__: config });
 		context.renderedNoscript = noscriptTemplate(config);
-		context.renderedExternals = `<script>(${serialize(headScript)})(window.__KV_CONFIG__);</script>`;
+		// eslint-disable-next-line max-len
+		context.renderedExternals = `<script>(${serialize(headScript)})(window.__KV_CONFIG__, ${serialize(oneTrustEvent)});</script>`;
```

Comment:
Inject oneTrustEvent code into head script

Reply:
The addition of this oneTrust transport script is a nice lightweight amount of overhead to ensure we can access that state in the embed.